frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -22,6 +23,7 @@ export const updateDashboardAndCards = createThunkAction(
   UPDATE_DASHBOARD_AND_CARDS,
   function () {
     return async function (dispatch, getState) {
+      const startTime = Date.now();
```

Comment:
What do you think about using `performance.now();` which should be more suitable for performance measurements?

Reply:
Yes, done [b126831](https://github.com/metabase/metabase/pull/35703/commits/b126831b5492e473cddeea4ae01da790927f0fa2)